app/views/formative_assessments/show.html.slim

```diff
@@ -1,30 +1,18 @@
-h1=@questionnaire.heading
-p=@questionnaire.content
+h1=translate_markdown(@questionnaire.heading)
```

Comment:
I have removed these tags when I'm rendering markdown because it'll likely not be valid semantic HTML that gets rendered. At least this makes sense for the paragraph tag and body content.